ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1071,7 +1071,6 @@ private class JdbcLedgerDao(
     (rejectionReason.description, rejectionReason match {
       case _: Inconsistent => "Inconsistent"
       case _: OutOfQuota => "OutOfQuota"
-      case _: TimedOut => "TimedOut"
```

Comment:
Removing this means loading an already stored rejection will fall through to the catch all case and fail with a `sys.error`. I think we need to keep the `TimedOut` case.

The alternative is to write a simple migration that updates all rejection reasons from `TimedOut` to `InvalidLedgerTime`.